fanficfare/dateutils.py

```diff
@@ -66,6 +66,28 @@ def parse_relative_date_string(reldatein):
     # discards trailing ' ago' if present
     m = re.match(relrexp,reldatein)
 
+    # If the date is displayed as Yesterday
+    if "Yesterday" in reldatein:
```

Comment:
Is there ever more to it than just `"Yesterday"`?

This shouldn't need to copy everything from below.  It should be possible to add just a "Yesterday" check, possibly by adding an `elif` to the `if m:` clause.

I'm thinking something like:
```
    value=None
    if m:
        value = m.group('val')
        unit_string = m.group('unit')
    elif "Yesterday" in reldatein:
        value = `1`
        unit_string = 'days'

    if value:
        unit = unit_to_keyword.get(unit_string)
        ...
```

(I honestly don't know why `value` is a string, then converted to `int`.  Possibly a py2 vs py3 thing...)